liferay-tango/liferay-portal
# LPS-171940 Update Select Scope User message

Motivation
=======
What is the problem this work solves, including
[Link to story or ticket](https://issues.liferay.com/browse/LPS-168141)
[Link to technical task or ticket](https://issues.liferay.com/browse/LPS-171940)

Proposed Solution
========
Change the Select Scope message, so the user knows that it can't be changed once the segment is saved.

Steps to Verify:
----------------
1. Activate FF LPS-166954
1. Go to People Segments
1. Create a new segment
1. The message _"Select the scope of your segment to specify where it can be used. It would not be possible to modify the scope after the segment is saved."_ should be visible

Screenshots (optional):
-----------------------
![Captura de pantalla 2022-12-30 a las 13 08 48](https://user-images.githubusercontent.com/5776822/210073741-47f639a0-3a26-477b-afdf-bce5a1dc58ef.png)
